### DISCUSSION TITLE
how much on average would it cost to run a GPT engineer for 1 hour  on GPT4 and GPT3.5?

### DISCUSSION THREAD

**Discussion by ravsau** [AntonOsika/gpt-engineer#162] [General]
trying to get a sense of how much GPT engineer will cost. Might be interesting to compare the price/performance with a local LLM model too

**Comment by patillacode** [comment]
Hi 

First of all, GPT Engineer is free and open source.

If you are maybe referring to how much would OpenAI API cost (which is what is used under the hood in this project), I would suggest you take a look at [their website](https://openai.com/pricing)

**Comment by ravsau** [comment]
Yup, I understand that GPT Engineer itself is free. I'm looking at the some data on actual OpenAI API cost per hour from others who have run GPT Engineer. 

I'm planning to run it myself sometime soon and will report my API costs here.

**Reply by RageshAntony** [reply]
@ravsau 

Have you ran and checked it?

**Comment by dabbinavo** [comment]
Just did a run for a very small project where about 5 files were generated:
![Screenshot_2023-07-21-15-55-09-45_40deb401b9ffe8e1df2f1cc5ba480b12~2.jpg](https://github.com/AntonOsika/gpt-engineer/assets/14122508/4f8a8ec4-2d8c-43b3-9e78-d92b57570bbe)
